fastlane/lib/fastlane/actions/register_devices.rb

```diff
@@ -14,7 +14,9 @@ def self.file_column_headers
       def self.run(params)
         platform = Spaceship::ConnectAPI::BundleIdPlatform.map(params[:platform])
 
-        if params[:devices]
+        if params[:devices_inv]
+          new_devices = params[:devices_inv].to_a
```

Comment:
for `devices_inv`, simply converting to array as there is no need to invert key-value

Reply:
Outdated after updating to Bool